.github/copilot-instructions.md

```
@@ -1,111 +1,36 @@
+dotnet format
+# build with warnings as errors and SquiggleCop baseline
+dotnet build /p:PedanticMode=true /p:SquiggleCop_AutoBaseline=true
+# run unit tests
+dotnet test --settings ./build/targets/tests/test.runsettings
+# optional: run benchmarks (requires local setup and manual selection)
+dotnet run --configuration Release --project tests/Moq.Analyzers.Benchmarks
+```
+
+Benchmarks are optional and may require additional local configuration.
+
+## Guidelines
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)